.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
you should do something like this:
https://github.com/NodeFactoryIo/js-libp2p-noise/blob/master/.github/workflows/publish.yml

butler-autotag has regex detection, so you should create .version file in root directory where we would update version
and ci would automatically, on merge to master, create tag, release and upload binaries to release